config/locales/analytics/benchmarking/content_base.yml

```diff
@@ -301,7 +301,7 @@ en:
           unadjusted_kwh: Unadjusted change (kWh)
           urn: URN
           variation_in_baseload_between_days_of_week: Variation in baseload between days of week
-          w_floor_area: w/floor area
+          w_floor_area: Watt/floor area
```

Comment:
Not sure if you've used that here, but there's a rake task to "Reinstall" the YML from the analytics. So suggest you merge you analytics branch, bump the tag and refresh the YML here.